gm4_mending_tanks/data/gm4/tags/functions/pulse_check.json

```diff
@@ -0,0 +1,6 @@
+{
+    "values":[
+        "mending_tanks:pulse_check"
```

Comment:
This module only has an init_check function, this pulse_check tag is not needed.